Tools/AP_Bootloader/board_types.txt

```diff
@@ -314,6 +314,15 @@ AP_HW_AIRVOLUTE_DCS2                 5200
 AP_HW_AOCODA-RC-H743DUAL             5210
 AP_HW_AOCODA-RC-F405V3               5211
 
+# IDs 5220-5239 reserved for UAV-DEV GmbH
```

Comment:
should put these down lower in numerical order to allow someone in the future to easily determine whats available